OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -79,6 +79,7 @@ public interface IOrderTargeter
 		int OrderPriority { get; }
 		bool CanTarget(Actor self, Target target, List<Actor> othersAtTarget, TargetModifiers modifiers, ref string cursor);
 		bool IsQueued { get; }
+		bool OverrideSelection();
```

Comment:
Why did you go for a method instead of a property? Do you foresee the need to have that method do something more elaborate than `return true|false`?